template/runtime/src/lib.rs

```diff
@@ -319,10 +319,14 @@ impl<F: FindAuthor<u32>> FindAuthor<H160> for FindAuthorTruncated<F> {
 
 const BLOCK_GAS_LIMIT: u64 = 75_000_000;
 const MAX_POV_SIZE: u64 = 5 * 1024 * 1024;
+/// The maximum storage growth per block in bytes (3 Mb).
+const MAX_STORAGE_GROWTH: u64 = 3 * 1024 * 1024;
+const GAS_LIMIT_STORAGE_GROWTH_RATIO: u64 = 15_000_000u64.saturating_div(MAX_STORAGE_GROWTH);
```

Comment:
After reading the description of your pr, I think this should be `75_000_000(BLOCK_GAS_LIMIT)` instead of `15_000_000u64` if I understand correctly.

Reply:
Yeah. I updated the test parameters